### PR TITLE
Added theming for TalkingUI

### PR DESCRIPTION
**Pull request by Krzmbrzl**
These styles will be used in the new TalkingUI feature.

### CODE REVIEW

File: Dark.qss [hunk 1]
```diff
@@ -1043,3 +1043,24 @@ AboutDialog QTextEdit {
...
+}
+
+TalkingUI [selected=false]:hover {
+  background-color: #333;
+}
+
+TalkingUI [selected=true] {
+  background-color: #3e4f5e;
+  border: 1px solid #3e4f5e;
+}
```
Comment (davidebeatrici) [inline Dark.qss:1066]: Keep newline at the bottom.

File: Lite.qss [hunk 2]
```diff
@@ -1043,3 +1043,24 @@ AboutDialog QTextEdit {
...
+}
+
+TalkingUI [selected=false]:hover {
+  background-color: #eee;
+}
+
+TalkingUI [selected=true] {
+  background-color: #dcedf5;
+  border: 1px solid #97b5c6;
+}
```
Comment (davidebeatrici) [inline Lite.qss:1066]: Keep newline at the bottom.

File: OSX Dark.qss [hunk 3]
```diff
@@ -1064,4 +1085,4 @@ QToolButton:on {
 
 QToolButton:on:pressed {
   background-color: transparent;
-}
+}
```
Comment (davidebeatrici) [inline OSX Dark.qss:1088]: Keep newline at the bottom.

File: OSX Lite.qss [hunk 4]
```diff
@@ -1064,4 +1085,4 @@ QToolButton:on {
 
 QToolButton:on:pressed {
   background-color: transparent;
-}
+}
```
Comment (davidebeatrici) [inline OSX Lite.qss:1088]: Keep newline at the bottom.

File: Dark.qss (hunk 1 above)
Comment (Krzmbrzl) [reply Dark.qss:1066]: This is actually how the preprocessor program thingy generated it... But I can add it manually :)

**Comment by Krzmbrzl** [comment]
@davidebeatrici I added the missing newlines.

Additionally I added a small note to the README as this was something I had a bit trouble understanding. Maybe it'll help someone in the future. I figured that I'd simply include it in this PR for the sake of simplicity :)

**Pull request merged**